core/ripemd160.js

```diff
@@ -30,6 +30,12 @@ sjcl.hash.ripemd160.hash = function (data) {
 
 sjcl.hash.ripemd160.prototype = {
     /**
+     * Maximal number of bits this function can hash.
+     * @constant
+     */
+    maxSafeLength: Math.pow(2, 53) - 1, //Number.MAX_SAFE_INTEGER
```

Comment:
why not just use the number here?
`9007199254740991`
Calculating this every time sjcl loads seems a lot of work. On the other hand we could also move it into a single place e.g. `sjcl.max_safe_integer = 9007199254740991`